plugins/teachplusplus/teachers/routes.php

```diff
@@ -5,50 +5,49 @@
 use Teachplusplus\Teachers\Models\Like;
 use Teachplusplus\Teachers\Models\Teacher;
 
-Route::group(['prefix' => 'api'], function() {
-
+Route::group(['prefix' => 'api'], function () {
     Route::get('teacher', function () {
-
         $teachers = Teacher::with('subjects', 'feedbacks.likes')->get();
         
         return $teachers;
     });
     
     Route::get('teacher/{id}', function ($id) {
-    
         $teacher = Teacher::with('subjects', 'feedbacks.likes')->findOrFail($id);
         
         return $teacher;
     });
 
     Route::post('feedback', function () {
         $data = request()->only([
```

Comment:
Naco su tu `$data`? Zda sa ze sa to uz neskor nepouziva

Reply:
Pre vytvorenie feedbacku.

Reply:
Aha vidim, som slepy. `$feedbackData` by bolo trosku zrozumitelnejsie.